ipaserver/install/ipa_restore.py

```diff
@@ -144,8 +144,10 @@ class Restore(admintool.AdminTool):
         paths.DNSSEC_TOKENS_DIR,
     ]
 
-    FILES_TO_BE_REMOVED = [
```

Comment:
Please leave ``FILES_TO_BE_REMOVED`` and just add ``FILES_TO_BE_CLEARED`` as additional list. We might need the remove feature in the future.